### DISCUSSION TITLE
No obs-plugins folder.

### DISCUSSION THREAD

**Discussion by EmeraldThunder1** [kounoike/obs-virtualbg#41] [Q&A] [answered]
> Extract the zip and copy the data and obs-plugins folders to your OBS Studio Folder (C:\Program Files\obs-studio by default).

There is no obs-plugins folder inside both the online repo and the folder I have downloaded. Am I doing somethign wrong?

**Comment by kounoike** [comment, marked as answer]
I think it is source-zip which you downloaded.
please download obs-virtualbg-v{VERSION}-win64.zip.
